editor.planx.uk/src/pages/FlowEditor/components/forms/FormModal.tsx

```diff
@@ -274,7 +274,7 @@ const FormModal: React.FC<{
                   makeUnique(id, parent);
                   navigate(rootFlowPath(true));
                 }}
-                disabled={disabled}
+                disabled={disabled || (isTemplatedFrom && isCustomisableNode(node))}
```

Comment:
When a node is tagged "customisation", we want it to be editable and you need to be able to "update" it, but you shouldn't be able to "delete" or "make unique" - these extra conditions account for that

Reply:
Open question here I suppose if we're happy with these buttons being disabled, or we could also completely hide them on the second condition !